depth.py

```diff
@@ -13,8 +13,12 @@ def depth(filename):
                 depth_map[name] += 1
             else:
                 depth_map[name] = 1
-    print(depth_map)
+    #print(depth_map)
 
+    with open('python_output.txt', 'w') as f: 
+        depth_items = depth_map.items()
+        for pair in depth_items:
+            f.write(f'{pair[0]} {pair[1]}\n')
```

Comment:
I think just `print`ing this this to stdout would be simpler. If you just do `print(f'...')` instead of opening a text file as output, it remains possible to *both* see the output in your console *or* write it to a file. That is, you can do this:

```
$ python3 depth.py something.gfa
```

to see the output with your eyes, or use shell redirection:


```
$ python3 depth.py > output.txt
```

to write it to `output.txt` for testing/comparison purposes. Leaving the output file non-hardcoded will make it easier to manipulate the output when necessary.